src/server.ts

```diff
@@ -227,6 +227,8 @@ const get_metadata = (ecosystem, name, version) => {
                 }
                 options['headers'] = {
                     'Authorization' : 'Bearer ' + config.api_token,
+                    'security_vendor' : 'snyk',
+                    'version' : '0.0.14',

```

Comment:
We shouldn't hardcode any vendor specific fields over here. Better pass these headers from plugin through env variable or other IPC mechanism recommended by vscode. You can look at how we are passing 3scale API endpoint and user key from plugin to lsp, which uses env variable based approach.